views/imprint/imprint.hbs

```diff
@@ -95,7 +95,7 @@
     <p>Ihre persönlichen Daten werden vom HPI geschützt. Das HPI gibt keine personenbezogenen Daten an Dritte weiter, es sei denn, die Weitergabe erfolgt in Erfüllung einer gesetzlichen Verpflichtung (z. B. im Rahmen von strafrechtlichen Ermittlungen). In diesem Falle werden nur solche Informationen weitergegeben, zu denen Ihre Schule und das HPI gesetzlich verpflichtet sind.</p>
     <p>Aus technischen oder organisatorischen Gründen behält sich das HPI vor, einzelne Hosting-Dienstleistungen an Subunternehmer zu vergeben. In diesem Fall werden die Nutzer/innen hierüber in Kenntnis gesetzt. Das HPI verpflichtet sich dazu, die hinzugezogenen Subunternehmer sorgfältig auszuwählen und so dafür Sorge zu tragen, dass es zu keiner Schwächung des Datenschutzniveaus kommt.</p>
     <p>Für die Nutzung interaktiver Inhalte externer Anbieter (z. B. Schulbuchverlage) werden Pseudonyme eingesetzt, die den Inhalte-Anbietern keinerlei Rückschlüsse auf die realen Nutzer/innen der HPI Schul-Cloud zulassen. Die pseudonymisierten Daten über die Nutzung digitaler Angebote (bspw. Testfragen) können von den Inhalte-Anbietern anonymisiert ausgewertet und zur Verbesserung ihrer Inhalte und Produkte eingesetzt werden.</p>
-    <p>Ein Verzeichnis aller Empfänger personenbezogener Daten finden Sie <a href="https://schul-cloud-hpi.s3.hidrive.strato.com/Willkommenordner/Datenschutz/Empfaenger-personenbezogener-Daten.pdf">hier</a>.</p>
+    <p>Ein Verzeichnis aller Empfänger personenbezogener Daten finden Sie <a href="https://s3.hidrive.strato.com/schul-cloud-hpi/Dokumente/Empfaenger.pdf">hier</a>.</p>
```

Comment:
please use constants defined in {{@root.theme.documents.baseDir}} here

Reply:
Is this already deployed in develop?

Reply:
ok, lets move that on the roadmap for later